components/common-styles/_custom-play.scss

```diff
@@ -0,0 +1,37 @@
+.custom-play__button {
+  @include appearance(none);
+  @include flex;
+  @include transform(scale(0.1));
+  @include transition(transform 250ms, opacity 250ms, background-color 250ms);
```

Comment:
This is sth. I stumbled upon when trying to make transition-transforms working across devices, even iOS:

Making use of a modified include called `transitionTransform` where the first attribute is the transform value, because -moz-transition will need -moz-transform as well, for example.
```
@mixin transition($transition...) {
  -webkit-transition: $transition;
     -moz-transition: $transition;
       -o-transition: $transition;
          transition: $transition;
}

// same as mixin transition but with prefixes for transform
// first argument must be transform, e.g.:
// @include transitionTransform(transform 1s, background-color 2s);
@mixin transitionTransform($transform...) {
  -webkit-transition: -webkit-#{$transform};
     -moz-transition: -moz-#{$transform};
       -o-transition: -o-#{$transform};
          transition: $transform;
}
```

Reply:
ah yeah, that makes sense. I'll make use of this, thanks!